libs/ballot-interpreter-nh-next/src/interpret.rs

```diff
@@ -339,10 +359,12 @@ pub fn interpret_ballot_card(side_a_path: &Path, side_b_path: &Path, options: &O
         front: InterpretedBallotPage {
             grid: front_grid,
             marks: front_scored_oval_marks,
+            normalized_image: make_normalized_image_buffer(front_image),
         },
         back: InterpretedBallotPage {
             grid: back_grid,
             marks: back_scored_oval_marks,
+            normalized_image: make_normalized_image_buffer(back_image),
```

Comment:
Serializing a relatively large image as JSON seems likely to take a _lot_ of memory and maybe be slow. Napkin math: 1700 pixels wide × 2200 pixels high × 4 bytes per pixel × 2 images is 29.9MB, which is our baseline that we can't really do better than without compression. Then we serialize it such that, on average, each byte becomes four bytes e.g. `0xff` (one byte) → `"255,"` (four bytes). That brings our total up to ~120MB. I'll do some testing, but I wanted to comment on this because it jumped out at me.

Reply:
I added some basic logging around the interpretation, serialization, and parsing and got this with `ts/src/interpret.test.ts`:

```
Interpretation took 79.192995ms
Serialization took 576.007626ms (size: 386871478)
  console.time
    parse JSON: 421 ms
```

It adds about 1s on my M1 MAX MBP and 386,871,478 bytes (387MB) worth of Rust string. I'm guessing this is because we're doing a pretty-print so there's lots of extra spaces per byte. Also, I think v8 is smart enough to keep the data encoded as UTF-8, but JS engines used to be(/ still sometimes are?) UTF-16 which would bring this up over 700MB!

Reply:
And yes, not doing pretty-printing sped it up and reduced the size significantly. That seems a worthwhile change even though in practice it'll be with a much smaller amount of data:

```
Interpretation took 78.782609ms
Serialization took 179.387241ms (size: 117576475)
  console.time
    parse JSON: 396 ms
```

Reply:
And if we don't serialize the normalized images as JSON:

```
Interpretation took 76.642562ms
Serialization took 107.414µs (size: 68016)
  console.time
    parse JSON: 0 ms
```

Reply:
This should be resolved by a8b2e60cb50ccc2b87bbc96815154c313f0a27e4.